schunk_gripper_driver/schunk_gripper_driver/driver.py

```diff
@@ -840,22 +847,24 @@ def _move_to_absolute_position_cb(
 
     def _grip_cb(
         self,
-        request: Grip.Request,
-        response: Grip.Response,
+        request,
```

Comment:
@PrajwalMangaluru
Is there a chance we could use some typing here? At least there's the `Any` type (`from typing import Any`). If the list is short, we could be explicit with `Grip.Request | GripGPE.Request`.